src/main/java/br/com/budgetboys/pf_finance_service/adapters/inbound/controller/income/IncomeController.java

```diff
@@ -0,0 +1,63 @@
+package br.com.budgetboys.pf_finance_service.adapters.inbound.controller.income;
+
+import br.com.budgetboys.pf_finance_service.adapters.outbound.service.income.IncomeService;
+import br.com.budgetboys.pf_finance_service.domain.income.IncomeCreateDTO;
+import br.com.budgetboys.pf_finance_service.domain.income.IncomeResponseDTO;
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("income")
```

Comment:
Só alterar para para o plural pois representa a coleção inteira.
```suggestion
@RequestMapping("incomes")
```